plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -456,9 +363,9 @@ def showControlInterface(self):
     ##  Send a command to printer. 
     #   \param cmd string with g-code
     def sendCommand(self, cmd):
-        if self.isPrinting():
+        if self._progress:
```

Comment:
The point is to make the API explicit. If you compare against a state, you know it is in that state. If you just compare against "magic values" you need to always figure out what those magic values are.